MsixCore/msixmgr/msixmgr.rc

```diff
@@ -124,6 +124,15 @@ BEGIN
     IDS_STRING_INVALID_ARCHITECTURE_ERROR "The deployment operation failed because the package targets the wrong processor architecture."
     IDS_STRING_PACKAGE_DOWNGRADE_ERROR "The package could not be installed because a higher version of this package is already installed."
     IDS_STRING_USER_CANCELLED_INSTALL_ERROR "User cancelled installation."
+    IDS_STRING_HELP_OPTION_UNPACK_CREATE "optional parameter that creates a new image with the specified -filetype and unpacks the packages to that image"
+    IDS_STRING_HELP_OPTION_UNPACK_ROOTDIRECTORY "root directory on an image to unpack packages to. Required parameter for unpacking to new and existing CIM files"
+    IDS_STRING_HELP_OPTION_UNPACK_FILETYPE "the type of file to unpack packages to. Valid file types include {VHD, VHDX, CIM}. This is a required parameter when unpacking to CIM files"
+    IDS_STRING_HELP_OPTION_MOUNTIMAGE "Mounts an image of the specified file type"
+    IDS_STRING_HELP_OPTION_MOUNTIMAGE_IMAGEPATH "the path to the image file to mount"
+    IDS_STRING_HELP_OPTION_MOUNTIMAGE_VOLUMEID "a GUID to associate with the mounted image. Use this GUID to unmount the mounted image"

```

Comment:
>GUID [](start = 101, length = 4)

be more specific of the format of the GUID, with or without curly braces, it looks like the code requires no curly braces?